app/src/main/java/de/danoeh/antennapod/activity/MediaplayerActivity.java

```diff
@@ -328,6 +328,7 @@ public boolean onPrepareOptionsMenu(Menu menu) {
         boolean isItemHasDownloadLink = media != null && (media instanceof FeedMedia) && ((FeedMedia) media).getDownload_url() != null;
         menu.findItem(R.id.share_download_url_item).setVisible(isItemHasDownloadLink);
         menu.findItem(R.id.share_download_url_with_position_item).setVisible(isItemHasDownloadLink);
+        menu.findItem(R.id.share_file).setVisible(((FeedMedia) media).fileExists());
```

Comment:
It did not crash the app (at least the one stream I tried). Have now added another check for null media